documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,160 @@
+
+- The API is serving slower-than-typical responses which occasionally time out
+- Openverse.org has a critical bug preventing search for mobile users using RTL
+  languages
+- A user has posted public comments using hateful language towards an Openverse
+  maintainer
+- Openverse.org is completely offline
+- Production resource usage is skyrocketing and could lead to downtime, but
+  users are not impacted yet
+
+All of these scenarios suggest that **Openverse behavior normally depended on or
+expected by regular users is not behaving as it should.**
+
+It is crucial to be proactive here; if something _might_ be an incident, but
+you’re not certain, it is better to record it than to ignore it. Do not assume
+"someone else" is handling an incident unless they have explicitly recorded it
+using the process defined here.
+
+## How do we _identify_ incidents?
+
+Incidents can be detected through various channels such as automated alerts,
+user reports, or through manual checks by maintainers. The Mini Support Rotation
+(MSR) runner makes these checks as part of their daily responsibilities.
+
+While the MSR runner should be monitoring our communication channels closely,
+**any online maintainer who sees something that might be an incident should
+follow the incident recording process outlined here.** This does not mean that
+the reporting maintainer is responsible for stabilizing the incident or acting
+as the incident lead. It simply means that they are responsible for making the
+initial report on the incident.
+
+## How do we _classify_ incidents?
+
+Incidents are classified in two ways:
+
+- **Severity** - How impactful the incident is
+- **Status** - The current state of the incident
+
+### Severity
+
+Incidents should be classified using one of three levels of severity. It is
+important to note that regardless of severity, working to stabilize services (or
+whatever harm mitigation is relevant for the incident) always **takes priority
+over regular project work and MSR duties.** Once service is stabilized, any
+ongoing investigations, preventative measures, or documentation changes will be
+prioritized by the team alongside our other work.
+
+This severity scale is intentionally distinct from our GitHub priority scale.
+Any GitHub issues created to _stabilize_ a Severity 1 or Severity 2 incident
+should be marked as "critical" in GitHub. Other issues should be prioritized at
+your own discretion.
+
+- **Severity 1**: Critical issue affecting all/most users and all/most
+  functionalities of Openverse.
+- **Severity 2**: Major issue affecting a large number of users or critical
+  functionality of Openverse.
+- **Severity 3**: Minor issue affecting a limited number of users or
+  non-critical functionality of Openverse.
+
+### Status
+
+- **Stabilization pending** - The incident is actively disrupting service
+- **Stabilized** - The incident is no longer directly disrupting service. root
+  causes may
```

Comment:
Looks like the sentence is incomplete here, maybe something like this was what followed (?)
```suggestion
- **Stabilized** - The incident is no longer directly disrupting service. Root
  causes may still be unknown.
```

Reply:
That would seem to overlap with `Under investigation` 🤔 I'm actually not sure what the difference between the two should be?

Reply:
I tried to add some clarifying language around that transition Staci, hopefully that helps!